katas/content/distinguishing_states/zero_plus/Placeholder.qs

```diff
@@ -0,0 +1,6 @@
+namespace Kata {
+    operation IsQubitZeroOrPlus (q : Qubit) : Bool {
+        // Implement your solution here...
+        return true;
+    }
+}
```

Comment:
Several of these files don't end with a newline. Best practice is to finish with a newline (hence why GitHub diffs flag them like this). Most editors adds this, or can be configured to.